git-tar/function/ops.go

```diff
@@ -647,3 +647,38 @@ func GitHubCloneURL(pushEvent sdk.PushEvent) (string, error) {
 
 	return cloneURL, nil
 }
+
+func checkCompatibleTemplates(stack *stack.Services, filePath string) (templatesErr error) {
```

Comment:
Please can you add a test for both methods?

Reply:
Yes I will 👍

Reply:
Done ✔️

Reply:
Added 👍